scripts/deploy.js

```diff
@@ -151,6 +152,19 @@ async function main() {
 	const plexusCoin = await (await PlexusCoin.deploy()).deployed();
 	console.log("PlexusCoin is deployed at: ", plexusCoin.address);
 
+	const airdrop = await deployWithProxy(
+        Airdrop, 
+        OwnableProxy, 
+        'Airdrop',
+		plexusCoin.address,
+		[]
+    );
+	console.log("Airdrop is deployed at: ", airdrop.address);
+
+	// Send 100000 PLX to the airdrop contract
+    const amount = ethers.utils.parseEther("100000");
+    plexusCoin.transfer(airdrop.address, amount);
```

Comment:
This is not necessary for now during deployment it's only needed during testing, so please comment it for now.